server/services/MailerService.ts

```diff
@@ -0,0 +1,69 @@
+import nodemailer, { Transporter, SentMessageInfo } from 'nodemailer';
+import { IMailerService } from 'types/services';
+import Utilities from 'server/util/Utilities';
+
+// @todo add ourEmail, emailUsername, emailPassword, and emailService as
+// environment variables when they become available. Temporary placeholders
+// provided until updated info available.
+export default class MailerService implements IMailerService {
```

Comment:
why do you need to implement IMailerService? its not like there are gonna be multiple class definitions following the same guidelines

Reply:
That's true, but the purpose of interfaces is not only to define shared types, but also to show a high level abstraction of a class. In most Object Oriented projects (such as C# and Java), it's generally considered a good practice to include the interface on services and I think it makes sense for NodeJS w/ TS, but I guess its really up to the FCC community. If you all believe I should get rid of it, I certainly will.